Fix:
bpgdec.c:339:9: error: implicitly declaring library function 'strrchr' with type 'char *(const char *, int)' [-Werror,-Wimplicit-function-declaration]
bpgdec.c:343:14: error: implicitly declaring library function 'strcasecmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]
--- bpgdec.c.orig	2018-04-21 04:42:55.000000000 -0500
+++ bpgdec.c	2021-11-05 05:00:32.000000000 -0500
@@ -27,6 +27,8 @@
 #include <math.h>
 #include <getopt.h>
 #include <inttypes.h>
+#include <string.h>
+#include <strings.h>
 
 /* define it to include PNG output */
 #define USE_PNG
